solution/backend/common/test_functions/common_functions.py

```diff
@@ -0,0 +1,5 @@
+import json
+
+
+def get_paginated_data(response):
```

Comment:
responses with pagination are returned as an ordereddict which makes comparing the data annoying.  So we convert it to a json object to make it easier to look at.